import/CoordinateConversion.qml

```diff
@@ -0,0 +1,327 @@
+/*******************************************************************************
+ *  Copyright 2012-2020 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+
+import QtQuick 2.12
+
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+
+/*!
+ * \qmltype CoordinateConversion
+ * \inqmlmodule Esri.ArcGISRuntime.Toolkit
+ * \since Esri.ArcGISRutime 100.8
+ * \brief The user interface for the coordinate conversion tool.
```

Comment:
Should we have a note that this file is shared for qml and cpp? Maybe that just confuses matters

Reply:
This is a good point. I was thinking that if we had a `CONTRIBUTING.md` file - we could go into more details on structure across the system there.